themes/login-forms/templates/Security.ss

```diff
@@ -11,13 +11,15 @@
         <% require css("silverstripe/login-forms: client/dist/styles/bundle.css") %>
     </head>
     <body>
-        <section>
+        <section class="log-in">
             <header>
-                <h1>
+                <h1 class="log-in__site-name">
                     $SiteConfig.Title
                     <% if not $SiteConfig.Title %>SilverStripe<% end_if %>
                 </h1>
             </header>
+            <% if $Message %><div class="log-in__message<% if $MessageType %> log-in__message--$MessageType<% end_if %>">$Message</div><% end_if %>
```

Comment:
Note that in SilverStripe 4.4 onwards we're using `$AlertType` for CMS related alerts (not directly in framework). This is an example where you might want to include that as well for 4.4 support